unime/src/i18n/de-DE/index.ts

```diff
@@ -255,6 +255,9 @@ const de = {
   },
   TIMELINE: {
     EMPTY: 'Noch keine Aktivitäten.',
+    ADDED_CREDENTIALS: 'TODO',
```

Comment:
Let's not do this as such things are easily forgotten compared to a type error when using the **_typesafe_**-i18n library. Touching a translation file without adding the actual translations should be avoided.

Reply:
ok no problem, I do think we should aim for strict typescript compiling, which means that no errors can exist in typing. Which would make this solution not possible, but for now I remove the fields again